lisk-dex-electron/src/PlaceOrder.jsx

```diff
@@ -266,6 +269,7 @@ export default class PlaceOrder extends React.Component {
 
   render() {
     const canTrade = this.context.keys[this.context.activeAssets[0]] && this.context.keys[this.context.activeAssets[1]];
+    const er = this.getEstimatedReturns();
```

Comment:
We should avoid using two letter variable names because they're hard to search for in the code.